addons/core/stringtable.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
```
What editor did you use?

Reply:
https://github.com/bux/tabler
and Sublime